pkg/volume/empty_dir/empty_dir.go

```diff
@@ -169,14 +169,14 @@ func (ed *emptyDir) SetUpAt(dir string) error {
 }
 
 func (ed *emptyDir) setupDefault(dir string) error {
-	return os.MkdirAll(dir, 0750)
+	return os.MkdirAll(dir, 0777)
```

Comment:
are there any real constants in this world?  today's 0777 might be tomorrow's 1666.